### PR TITLE
feat: Add Kafka destination

### PR DESCRIPTION
**Pull request by yevgenypats**
This adds the long time requested Kafka destination :)

Currently it supports SASL authentication and in general Kafka client can be super configurable but this should be good for starters and then we can wait for user input regarding more authentication options or other options that we would like to expose from the underlying Kafka client.

### CODE REVIEW

File: plugins/destination/kafka/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,465 @@
+# Changelog
```
Comment (erezrokah) [inline plugins/destination/kafka/CHANGELOG.md:1]: Let's delete this file altogether. Release please (our release bot) creates it

File: plugins/destination/kafka/README.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# CloudQuery Kafka Destination Plugin
+
+This destination plugin lets you sync data from a CloudQuery source to a PostgreSQL compatible database.
+
+## Links
+
+- [User Guide](https://cloudquery.io/docs/plugins/destinations/postgresql/overview)
```
Comment (erezrokah) [inline plugins/destination/kafka/README.md:7]:
```suggestion
- [User Guide](https://cloudquery.io/docs/plugins/destinations/kafka/overview)
```

File: website/pages/docs/plugins/destinations/kafka/overview.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+```yaml
+kind: destination
+spec:
+  name: "kafka"
+  path: "cloudquery/kafka"
+  version: "VERSION_DESTINATION_KAFKA"
+  write_mode: "append" # only supports 'append'
+
+  spec:
+    brokers: ["localhost:29092"]
```
Comment (erezrokah) [inline website/pages/docs/plugins/destinations/kafka/overview.md:25]:
```suggestion
    brokers: ["<broker-host>:<broker-port>"]
```

Not sure, is the port optional? Can this be a URL, or it has to be <host:port>?

**Review by erezrokah — CHANGES_REQUESTED** [review]
Looks good, added a few comments

File: website/pages/docs/plugins/destinations/kafka/overview.md (hunk 3 above)
Comment (yevgenypats) [reply website/pages/docs/plugins/destinations/kafka/overview.md:25]: `host:port`

**Review by erezrokah — APPROVED** [review]

**Pull request merged**